pyproject.toml

```diff
@@ -39,6 +39,13 @@ ruff = "^0.0.261"
 toml-sort = "^0.23.0"
 types-requests = "^2.28.11.17"
 
+[tool.poetry.group.sphinx-deps.dependencies]
+python = ">=3.9.0,<4.0"
```

Comment:
I think this works, but just double-check that we can still install the main package with python 3.8. The easiest way might be to temporarily comment out this line in the github action (https://github.com/groundlight/python-sdk/blob/main/.github/workflows/cicd.yaml#L89), and see it run with python 3.7-3.11.

Reply:
Verified!